src/arrays.jl

```diff
@@ -237,4 +238,11 @@ immutable ArrayInterfaceError{T<:AbstractString} <: DatabaseError{ColumnarArrayI
     message::T
 end
 
+if VERSION >= v"0.5-"
```

Comment:
best to be more specific about VERSION dependent cutoffs like this, or use isdefined